Jenkinsfile

```diff
@@ -144,11 +154,11 @@ pipeline {
     stage('h2 UNIT, engine IT, webapp IT') {
       parallel {
         stage('db-UNIT-h2') {
-          when {
-            expression {
-              cambpmWithLabels('h2', 'rolling-update', 'migration', 'all-db', 'default-build', 'authorizations')
-            }
-          }
+//          when {
```

Comment:
You commented out a lot of label expressions. Why?

Reply:
They are for testing purposes to run the stages without labels. The commits will be removed before merge.